gurobi_sphinxtheme/theme/static/gurobi-tabs.css

```
@@ -0,0 +1,74 @@
+/* Adjust sphinx-tabs style to be a bit more like sphinx-inline-tabs */
+
+.sphinx-tabs-tab {
+    padding-top: 0.1rem;
+    padding-bottom: 0.2rem;
+    padding-left: 1rem;
+    padding-right: 1rem;
+    font-family: var(--font-stack);
+    font-size: 0.875em;
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    border-bottom: 2px solid;
+    margin: -1px;
+    border-color: transparent;
+}
+body[data-theme="dark"] .sphinx-tabs-tab {
+    background-color: transparent;
+    border-color: transparent;
+}
+
+.sphinx-tabs-tab[aria-selected="true"] {
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    border-bottom: 2px solid;
+    margin: -1px;
+    border-color: var(--color-brand-secondary);
+}
+body[data-theme="dark"] .sphinx-tabs-tab[aria-selected="true"] {
+    background-color: transparent;
+}
+
+.sphinx-tabs-tab:hover {
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    border-bottom: 2px solid;
+    margin: -1px;
+    border-color: var(--color-foreground-secondary);
+}
+body[data-theme="dark"] .sphinx-tabs-tab:hover {
+    border-color: var(--color-foreground-secondary);
```

```suggestion
    border-color: var(--color-brand-content);
```
I think this looks nicer, but I may have missed why you set this.

---

Thanks! Was not familiar with that one, I guess I assumed it would be a garish red ...